systemtest/src/test/java/io/strimzi/systemtest/specific/SpecificST.java

```diff
@@ -213,9 +226,10 @@ void testRackAwareConnectWrongDeployment() {
         KafkaConnectUtils.sendReceiveMessagesThroughConnect(kcPods.get(0), TOPIC_NAME, kafkaClientsPodName, NAMESPACE, clusterName);
 
         // Revert changes for CO deployment
-        ResourceManager.setClassResources();
-        BundleResource.createAndWaitForReadiness(BundleResource.clusterOperator(NAMESPACE).build());
-        ResourceManager.setMethodResources();
+        // TODO: how to solve this...???
+//        ResourceManager.setClassResources();
+        resourceManager.createResource(sharedExtensionContext, BundleResource.clusterOperator(NAMESPACE).build());
+//        ResourceManager.setMethodResources();
```

Comment:
Solved or?

Reply:
It's solved by sharedExtensionContext, which ensures that the resource manager will delete resource if and only if @AfterAll